proto/mixer.proto

```diff
@@ -252,6 +253,44 @@ message GetPlacesInResponse {
   string payload = 1;
 }
 
+// Request to get related places of a place for a given statistical variable.
+message GetRelatedPlacesRequest {
+  // Place DCIDs.
+  repeated string dcids = 1;
+
+  // Population type.
+  string population_type = 2;
+
+  // A list of constraining property values that defines the population.
+  repeated PropertyValue pvs = 3;
+
+  // The measured property of the observation.
+  string measured_property = 4;
+
+  // (Opitonal) Measurement method.
+  string measurement_method = 5;
+
+  // (Opitonal) Measurement denominator.
+  string measurement_denominator = 6;
+
+  // (Opitonal) Measurement qualifier.
+  string measurement_qualifier = 7;
+
+  // (Opitonal) Scaling factor.
+  string scaling_factor = 8;
+
+  // (Optional) Unit.
+  string unit = 9;
+
+  // The statistics type, like "measured", "median", etc...
```

Comment:
Should we make it optional with default being "measuredValue"?

(in some other context, we were talking about it right?)

Reply:
Good idea - will change it when the BT has StatType (now BT doesn't include StatType).